backend/src/routes/notes.ts

```diff
@@ -24,28 +41,43 @@ router.get("/notes/:id", async (req: Request, res: Response, next: NextFunction)
   }
 });
 
+/**
+ * * This route will update the answers of a single note document.
+ * @param id: ObjectID of the notes document to be retreived.
+ * @body The body should be a JSON in the form:
+ * [
+    {
+        question_id: "the hashed ID 1",
+        type: "bullet",
+        answer: "updated answer"
+    },
+    {
+        question_id: "the hashed ID 2",
+        type: "bullet",
+        answer: ["new answer1", "new answer2"]
+    }
+]
+ * @response "Success" with new, updated note if successfully updated, "Invalid" otherwise.
+ */
 type UpdateNoteRequestBodyType = Infer<typeof UpdateNoteRequestBodyCake>;
 router.patch(
   "/notes/:id",
   validateReqBodyWithCake(UpdateNoteRequestBodyCake),
   async (req: Request, res: Response, next: NextFunction) => {
     try {
-      console.log("Patching...");
       const documentId = req.params.id;
       const updatedNotes: UpdateNoteRequestBodyType = req.body;
       await updateNotes(updatedNotes, documentId);
-      console.log("##################");
       const noteDoc = await Note.findById(documentId);
-      console.log(noteDoc?.answers);
       return res.status(200).json({
```

Comment:
nice work! Can you please update all the status codes used for these routes on the API sheet?
https://docs.google.com/spreadsheets/d/1h1LmmF9NtEkucmPMpy8WkK-ubjwUXb_qWc0b0USPN04/edit#gid=0